addons/payment_asiapay/models/payment_provider.py

```diff
@@ -17,6 +17,14 @@ def _domain_asiapay_currency_id(self):
     code = fields.Selection(
         selection_add=[('asiapay', "AsiaPay")], ondelete={'asiapay': 'set default'}
     )
+    asiapay_brand = fields.Selection(
+        string="AsiaPay Brand",
+        help="The brand associated to your AsiaPay account.",
+        selection=[('paydollar', 'PayDollar'), ('pesopay', 'PesoPay'),
+                    ('siampay', 'SiamPay'), ('bimopay', 'BimoPay')],
```

Comment:
```suggestion
        selection=[('paydollar', "PayDollar"), ('pesopay', "PesoPay"),
                    ('siampay', "SiamPay"), ('bimopay', "BimoPay")],
```

Strings that are read by humans are written inside `"` instead =]
Sorry, should have seen it sooner